src/components/activity/ActivityList.js

```diff
@@ -0,0 +1,60 @@
+import styled from 'styled-components';
+
+function ActivityList({activities, onHandleDelete}) {
+  const handleDelete = event => {
+    const activityId = event.target.dataset.id;
+    onHandleDelete(activityId);
+  };
+  return (
+    <>
+      <div>
+        <label>Aufgaben des Tages</label>
+      </div>
+      {activities.map(activity => (
+        <div key={activity.id}>
+          <StyledActivitiList>
+            <label>Dein Tag:</label>
+            <p>{activity.activity}</p>
+            <p>{activity.startTime}</p>
+            <p>{activity.endTime}</p>
+            <StyledButton onClick={handleDelete} data-id={activity.id}>
+              Eintrag löschen
+            </StyledButton>
+          </StyledActivitiList>
+        </div>
+      ))}
+    </>
+  );
+}
+
+const StyledActivitiList = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  margin: 1rem 0 0.25rem 0;
```

Comment:
You use some duplicate properties (e.g. `margin`, `border`...), that's why the linter check fails (see https://github.com/Ava-VictoriaCrivello/Capstone-Projekt/runs/7669698617?check_suite_focus=true) 
Please check against the error message and remove duplicate properties.